ibis/expr/operations/relations.py

```diff
@@ -330,6 +340,12 @@ class DatabaseTable(PhysicalTable):
     namespace: Namespace = Namespace()
 
 
+@public
+class VersionedDatabaseTable(DatabaseTable):
```

Comment:
So `at_time` could be placed here, and the `JoinKind` literal could have a new member to mark temporal joins.

Reply:
Thanks, made the suggested changed.